Release6.md

```diff
@@ -0,0 +1,277 @@
+
+### Preview: Domain Manager Persona
+
+A Domain Manager role has been established in a [standard draft in SCS](https://docs.scs.community/standards/scs-0302-v1-domain-manager-role) aiming to allow self-service capabilities for customers at the identity API.
+Work is in progress to [contribute this functionality to OpenStack Keystone](https://bugs.launchpad.net/keystone/+bug/2045974) and the corresponding [upstream spec](https://review.opendev.org/c/openstack/keystone-specs/+/903172) is currently under review.
+The feature is expected to be available in the next SCS release.
+
+### Preview: Central API
+
+To improve the experience of SCS cloud customers, the idea of a "Central API" was discussed. Such API should enable customers to manage various "as-a-Service" resources. For example: OpenStack instances as well as Kubernetes clusters as well as Keycloak OAuth2 clients.
+Read about the tradeoffs and ideas in the [central-api repository](https://github.com/SovereignCloudStack/central-api) and feel free to test out the [POC](https://github.com/SovereignCloudStack/central-api/blob/0422e8ca24667b04b7364ffa461f85c3de51a50a/poc-setup.md).
+
+### Preview: Keycloak Home-IdP-discovery
+
+To improve flexibility of onboarding new customer domains via IdP federation, SCS now deploys Keycloak with the [Home-IdP-discovery](https://github.com/sventorben/keycloak-home-idp-discovery) plugin. The idea is, that Keystone federates out to a single Keycloak "proxy" realm (called `osism` in the testbed) and using the plugin, Keycloak can identify the user specific realm from an email-format login-ID. Operators can create dedicated realms for each customer and Keycloak uses internal federation to redirect from the "proxy" realm to the specific customer realm. In the customer ream they can configure IdP federation (OpenID-Connect or SAML) to their own IAM solution. The [IAM section](https://docs.scs.community/docs/iam) of the SCS documentation shall be extended to detail the configuration. SCS is working upstream to contribute required enhancements in the
+mapping of users, groups and roles from OpenID-Connect token claims to the OpenStack Keystone access management.
+
+
+### Upcoming: automated pentesting security pipeline
+
+In the beginning of 2024 we started with the conception and implementation of an automated security pipeline. This pipeline will serve as a security scanning tool that is either triggered by deployments or due to manual activation in your infrastructure. The pipeline will contain six tools: Naabu, httpx, Nuclei, Greenbone (Community Edition), ZAP and DefectDojo. While the first three are meant to discover network-related issues like open ports and attack vectors, GCE and ZAP will deliver deeper security inspections of hosts in your SCS environment. Reports are collected in DefectDojo, allowing an overview over security-related changes and seeing the current state. The pipeline will be available as a platform tool in R7.
```

Comment:
I'm not really happy with this. These are technical release notes and we should focus on the stuff we are releasing. This seems good content for an outlook blogpost imho.

Reply:
Well, maybe we shorten this and attach it to the section that talks about the pentesting that already happened and that improved R6 security?